pcdsdevices/beam_stats.py

```diff
@@ -26,3 +28,63 @@ class SxrGmd(Device):
 
     def __init__(self, prefix='', name='sxr_gmd', **kwargs):
         super().__init__(prefix=prefix, name=name, **kwargs)
+
+
+class BeamEnergyRequest(PVPositionerIsClose):
+    """
+    Positioner to request beam color changes from ACR in eV.
+
+    It is up to ACR how to and whether to fulfil these requests. This is often
+    fulfilled by moving the Vernier but can also be a more involved process.
+
+    Motion is considered "done" when the photon energy readback is close enough
+    to the requested value. The default tolerance here is 30 eV, but this can
+    be changed on a per-instance basis by passing atol into the init, or on a
```

Comment:
```suggestion
    be changed on a per-instance basis by passing ``atol`` into the initializer, or on a
```

Reply:
I think this rides me over an 80-length line, I'll make the equivalent change in a different commit